src/javascripts/routes.js

```diff
@@ -4,10 +4,15 @@ import {
   IndexRoute
 } from 'react-router'
 import App       from './presenters/application'
+import Blog from './components/blog/Blog'
+import Article from './components/blog/article/Article'
 import Chat from './components/Chat'
+import Author from './components/blog/author/AuthorBio'
 
 export default (
-  <Route path="/" component={ App }>
-    <IndexRoute component={ Chat } />
+  <Route path="/" component={App}>
+    <IndexRoute component={Blog} />
+    <Route path="/article" component={Article} />
+    <Route path="/author" component={Author} />
```

Comment:
In order to load different articles/authors using the same templates, these will look more like this:
`<Route path="/author/:id" component={Author} />`

We'll get to using that `id` later